README.md

```diff
@@ -24,6 +24,28 @@ This library requires:
 
 This package provides a Nuke plugin that creates jobs for AWS Deadline Cloud using the [AWS Deadline Cloud client library][deadline-cloud-client]. Based on the loaded comp it determines the files required, allows the user to specify render options, and builds an [OpenJD template][openjd] that defines the workflow.
 
+### Getting Started
+
+To install the submitter manually, you can use pip.
+
+```sh
+$ pip install deadline-cloud-for-nuke -t target-directory
+```
+
+For Windows,
+```sh
+$ set NUKE_PATH=%NUKE_PATH%;<target-directory>/deadline/nuke_submitter
+```
+
+For Linux and MacOS:
+```sh
+$ export NUKE_PATH=${NUKE_PATH}:<target-directory>/deadline/nuke_submitter
+```
+
+After installation and within the same terminal, run `Nuke<version>` executable. The Nuke submitter should now be available in the AWS Deadline menu.
```

Comment:
It'd be helpful to give an example command here, especially for Macs where it's less obvious.

Reply:
I will add this on the separate PR! 🙏  Thanks for the suggestion.